src/common/common-types.ts

```diff
@@ -89,6 +90,7 @@ export interface NodeData {
     readonly isDisabled?: boolean;
     readonly isLocked?: boolean;
     readonly inputData: InputData;
+    readonly outputData?: OutputData;
```

Comment:
I don't think this is a good idea. Nothing about output data is an intrinsic property of a node. These outputs are values computed from nodes. Just like types, I think output values should live outside of nodes and be tracked separately.